packages/core/src/modules/routing/RecipientModule.ts

```diff
@@ -264,66 +264,34 @@ export class RecipientModule {
     return event.payload.mediationRecord
   }
 
-  public async provision(mediatorConnInvite: string) {
-    this.logger.debug('Provision Mediation with invitation', { invite: mediatorConnInvite })
-    // Connect to mediator through provided invitation
-    // Also requests mediation and sets as default mediator
-    // Assumption: processInvitation is a URL-encoded invitation
-    const invitation = await ConnectionInvitationMessage.fromUrl(mediatorConnInvite)
-
-    // Check if invitation has been used already
-    if (!invitation || !invitation.recipientKeys || !invitation.recipientKeys[0]) {
-      throw new AriesFrameworkError(`Invalid mediation invitation. Invitation must have at least one recipient key.`)
-    }
-
-    let mediationRecord: MediationRecord | null = null
-
-    const connection = await this.connectionService.findByInvitationKey(invitation.recipientKeys[0])
-    if (!connection) {
-      this.logger.debug('Mediation Connection does not exist, creating connection')
-      // We don't want to use the current default mediator when connecting to another mediator
-      const routing = await this.mediationRecipientService.getRouting({ useDefaultMediator: false })
-
-      const invitationConnectionRecord = await this.connectionService.processInvitation(invitation, {
-        autoAcceptConnection: true,
-        routing,
-      })
-      this.logger.debug('Processed mediation invitation', {
-        connectionId: invitationConnectionRecord.id,
-      })
-      const { message } = await this.connectionService.createRequest(invitationConnectionRecord)
-      const outbound = createOutboundMessage(invitationConnectionRecord, message)
-      await this.messageSender.sendMessage(outbound)
-
-      const completedConnectionRecord = await this.connectionService.returnWhenIsConnected(
-        invitationConnectionRecord.id
-      )
-      this.logger.debug('Connection completed, requesting mediation')
-      mediationRecord = await this.requestAndAwaitGrant(completedConnectionRecord, 60000) // TODO: put timeout as a config parameter
-      this.logger.debug('Mediation Granted, setting as default mediator')
-      await this.setDefaultMediator(mediationRecord)
+  /**
+   * Connects to mediator through provided invitation and requests mediation and sets
```

Comment:
Outdated comment, this doesn't connect to the mediator anymore

Reply:
Updated.